nuitka/tools/testing/Virtualenv.py

```diff
@@ -37,6 +37,7 @@ class Virtualenv(object):
     def __init__(self, env_dir):
         self.env_dir = os.path.abspath(env_dir)
 
+    """
```

Comment:
I've commented out the entire `runCommand` for testing purposes, but everything still runs correctly for some reason...